tasks/stop-service.yml

```diff
@@ -34,6 +34,11 @@
       until: jenkins_service_running is failed
       retries: 60
       delay: 1
+
+    # Again, a paranoid measure aimed at preventing potential rsync failures.
```

Comment:
Similarly, I don't see why we mention rsync here.

Reply:
Fixed in 7d8ade8.